C-Plus-Plus/ds/alternate_merging_of_linked_list.cpp

```diff
@@ -0,0 +1,150 @@
+/*
+
+Author: Mohim Singla
+
+
+C++ program to merge alternate nodes two linked lists.
+
+*/
+#include <iostream>
```

Comment:
Great work! You need to add comments, that would really be a great contribution. 
Make sure you add comments above the code lines and not alongside.